packages/paste-core/components/slider/src/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import type {UIStates, BackgroundColorTokens} from './types';
+
+export const getColorToken = (states: UIStates): BackgroundColorTokens => {
```

Comment:
question/suggestion: How hard would it be to do a subtle animation on/off hover so the color change isn't as drastic?

Reply:
very! Done.